### PR TITLE
(VDB-1540) Add new auction contracts to beta

### PR DESCRIPTION
**Pull request by rmulhol**
- [X] Flap
- [X] Flip BAT-A
- [X] Flip ETH-A
- [x] Flip KNC-A
- [x] Flip MANA-A
- [x] Flip TUSD-A
- [X] Flip USDC-A
- [X] Flip USDC-B
- [x] Flip WBTC-A
- [x] Flip ZRX-A
- [X] Flop

### CODE REVIEW

File: transformers/events/DOCUMENTATION.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ If the contract isn't already present in the environment you'll need to add it b
    
     ``` toml
     [contract]
-    [contrac t.MCD_FLIP_ETH_A]
+    [contrac t.MCD_FLIP_ETH_A_1.0.0]
```
Comment (paytonrules) [inline transformers/events/DOCUMENTATION.md]: Why is there a space between c and t here?

Comment (rmulhol) [reply transformers/events/DOCUMENTATION.md]: yikes, there shouldn't be!

Comment (paytonrules) [reply transformers/events/DOCUMENTATION.md]: At least it's just docs.

File: transformers/storage/median/initializers/median_mana/initializer.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package median_mana
+
+import (
+	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/median/initializers"
+)
+
+var StorageTransformerInitializer = initializers.GenerateStorageTransformerInitializer(constants.GetContractAddress("MEDIAN_MANA"))
```
Comment (paytonrules) [inline transformers/storage/median/initializers/median_mana/initializer.go:8]: ~Shouldn't this be "MEDIAN_MANA_100"~ never mind.

**Review by paytonrules — DISMISSED** [review]

File: transformers/integration_tests/rely_test.go [hunk 3]
```diff
@@ -22,58 +22,124 @@ var _ = Describe("Rely transformer", func() {
...
 	})
 
-	Context("Flip ETH rely events", func() {
+	Context("Flap v1.0.9 rely events", func() {
+		usrAddress := "0xBE8E3e3618f7474F8cB1d074A26afFef007E98FB"
+		msgSenderAddress := "0xdDb108893104dE4E1C6d0E47c42237dB4E617ACc"
+		relyIntegrationTest(int64(10510870), test_data.FlapV109Address(), msgSenderAddress, usrAddress)
+	})
+
+	Context("Flip BAT v1.0.9 rely events", func() {
```
Comment (elizabethengelman) [inline transformers/integration_tests/rely_test.go:43]: wonder if we'd want a test for BAT v1.0.0 rely events too - but i don't think that's a huge deal, and def not merge blocking!

**Review by elizabethengelman — DISMISSED** [review]

**Review by elizabethengelman — APPROVED** [review]

**Pull request merged**